frontend/src/routes/Governance/discovered/ByCluster/DiscoveredByClusterPage.tsx

```diff
@@ -0,0 +1,79 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { useParams } from 'react-router-dom-v5-compat'
+import { AcmAlert, AcmPage, AcmPageHeader } from '../../../../ui-components'
+import { NavigationPath } from '../../../../NavigationPath'
+import { useMemo } from 'react'
+import { useTranslation } from '../../../../lib/acm-i18next'
+import Grid from '@mui/material/Grid'
+import { getEngineWithSvg } from '../../common/util'
+import { Box } from '@mui/material'
+import { useFetchPolicies } from '../useFetchPolicies'
+import { EmptyState, EmptyStateIcon, PageSection, Spinner, Title } from '@patternfly/react-core'
+import DiscoveredByCluster from './DiscoveredByCluster'
+
+export default function DiscoveredByClusterPage() {
+  const params = useParams()
+  const policyKind = params.kind ?? ''
+  const policyName = params.policyName ?? 'Unknown policyName'
+  const apiGroup = params.apiGroup ?? 'Unknown apiGroup'
```

Comment:
These default strings shouldn't be sent to `useFetchPolicies` since they could be used in the search query.